components/DashboardPage/components/HabitTracker/HabitsList.tsx

```diff
@@ -4,20 +4,38 @@ import { Habit } from './Habit';
 
 interface Habit {
   label: string;
+  days: boolean[];
 }
 
 export function HabitsList() {
   const [habitsList, setHabitsList] = useState<Habit[]>([
     {
-      label: 'Habit 1',
+      label: '',
+      days: [
+        false,
+        false,
+        false,
+        false,
+        false,
+        false,
+        false,
+      ],
     },
   ]);
 
   return (
     <Stack vertical>
       <Stack vertical spacing="tight">
-        {habitsList.map((habit) => (
-          <Habit key={habit.label} label={habit.label} />
+        {habitsList.map((habit: Habit, index: number) => (
+          <Habit
+            key={index}
```

Comment:
same here, `key=index` is not good practice.
here's what chatGPT thinks: 


_This is because the index of an item can change if items are added or removed from the list, which can cause React to re-render the entire list.
Using the index of an item as the key can also lead to other problems, such as inconsistencies when sorting the list or when filtering items out of the list. In general, it is better to use a unique and stable identifier for each item in the list as the key, such as a unique ID that is assigned to each item when it is added to the list._